src/js/player.js

```diff
@@ -2062,6 +2070,41 @@ class Player extends Component {
   }
 
   /**
+   * Returns the current source objects.
+   *
+   * @return {Object[]} The current source objects
+   * @method currentSources
+   */
+  currentSources() {
+    const source = this.currentSource();
+    const sources = [];
+
+    // assume `{}` or `{ src }`
+    if (Object.keys(source).length !== 0) {
```

Comment:
Agreed. I don't know if it's merged yet, but I introduced `utils/obj` in a PR recently. An `isEmpty` could be useful there. If not, this is fine, I think.